doc/api/cli.md

```diff
@@ -990,7 +990,7 @@ added:
 
 > Stability: 1.0 - Early development
 
-If the `--experimental-default-config-file` flag is present, Node.js will look for a
+Unless `--no-experimental-default-config-file` flag is present, Node.js will look for a
```

Comment:
This is still an experimental feature (line 991). What's the goal of this change? We should also make it stable at the same time, or not enable it this early.